docs/reference/ImageDraw.rst

```diff
@@ -243,6 +243,7 @@ Methods
 .. py:method:: ImageDraw.line(xy, fill=None, width=0, joint=None)
 
     Draws a line between the coordinates in the ``xy`` list.
+    The coordinate pixels are included in the drawn line.
```

Comment:
Is this a bit clearer?
```suggestion
    The endpoint coordinate pixels are included in the drawn line.
```
Or simply?
```suggestion
    The endpoint pixels are included in the drawn line.
```

Reply:
Feel free to update it with any wording that you think is clearer, but both `line()` and `polygon()` accept a series of coordinates. I wonder if saying 'endpoint' implies it is only the first and last of those coordinates that are included?

Reply:
Hmm, good point. Not sure now! Let's stick with the original :)